analyzers/rspec/cs/S5753_c#.html

```diff
@@ -28,23 +32,24 @@ <h2>Compliant Solution</h2>
 {
   ...
 }
-or
+</pre>
```

Comment:
for this, I removed the `or` from the `{code}` space (it doesn't compile)

Reply:
Removing or commenting out `// or` would do the same trick, right?

There's the same issue with VB, isn't it?

Reply:
yes, it's the same change for VB (line 34 in the VB HTML)